app/views/hosts/multiple_disassociate.html.erb

```diff
@@ -1,5 +1,12 @@
-<%= render 'selected_hosts', :hosts => @hosts %>
+<% non_physical_hosts = @hosts.with_compute_resource %>
+<% physical_hosts = @hosts.to_a - non_physical_hosts.to_a %>
+<%= render 'selected_hosts', :hosts => non_physical_hosts %>
 
 <%= form_tag update_multiple_disassociate_hosts_path({:host_ids => params[:host_ids]}) do %>
   <small><%= _('Disassociate the selected hosts from their VMs') %></small>
+  <br>
+  <% search = '' %>
+  <% physical_hosts.each { |host| search += "name = #{host.name} OR " } %>
+  <%= _('The list is excluding %s ') % (@hosts.count - non_physical_hosts.count) %>
```

Comment:
Someone marked the hosts and suddenly doesn't see them, if he has a list of 20 it's hard to find where he accidentally marked a physical host so this way he can see which hosts are not there.